onnxruntime/__init__.py

```diff
@@ -44,6 +44,15 @@
 except ImportError:
     pass
 
+try:
```

Comment:
I would prefer to have a mechanism only available when onnxruntime is compiled for this EP and not something always in the code whatever the compilation are.

Reply:
Hello @xadupre!
Could you take a look at the changes? Maybe you also had some ideas about this?

Reply:
It looks ok. At least, users know this code is only needed for this provider. I had in mind something which does not add extra code when onnxruntime is compiled without this provider. Something like an import made from C++ surrounded by #ifdef ... STVM ... #endif.